File: exercises/D111/solution.pas
var
    w, h, bmi: double;
begin
    readln(w, h);
    bmi := w / (h * h);
    writeln(bmi:0:3);
    if (bmi < 18.5) then
        writeln('Underweight')
    else if (bmi < 23.0) then
        writeln('Normal')
    else if (bmi < 25.0) then
        writeln('Overweight')
    else writeln('Obese');
end.

File: exercises/D111/solution.c
#include <stdio.h>
int main() {
    double w, h;
    scanf("%lf%lf", &w, &h);
    double bmi = w / (h * h);
    printf("%.3lf\n", bmi);
    if (bmi < 18.5) {
        printf("Underweight\n");
    }
    else if (bmi < 23.0) {
        printf("Normal\n");
    }
    else if (bmi < 25.0) {
        printf("Overweight\n");
    }
    else {
        printf("Obese\n");
    }
    return 0;
}
